packages/contracts/deploy-config/mainnet.json

```diff
@@ -44,6 +44,11 @@
   "governorProposalThreshold": 1,
   "governorVotesQuorumFractionPercent": 51,
   "timeLockMinDelaySeconds": 2592000,
+  "l2GovernorVotingDelayBlocks": 0,
+  "l2GovernorVotingPeriodBlocks": 129600,
+  "l2GovernorProposalThreshold": 1,
+  "l2GovernorVotesQuorumFractionPercent": 51,
+  "l2TimeLockMinDelaySeconds": 604800,
```

Comment:
Almost all of the config is the same as the L1 configs. There is no need to separate them.